kstreamplify-core-test/src/main/java/com/michelin/kstreamplify/KafkaStreamsStarterTest.java

```diff
@@ -84,14 +84,23 @@ protected Instant getInitialWallClockTime() {
         return Instant.ofEpochMilli(1577836800000L);
     }
 
+    /**
+     * Default storage path for the storage directory.
+     *
+     * @return The default storage path
+     */
+    protected String getStoragePath() {
```

Comment:
Do you think we can grant the user the possibility to give `Properties` instead?

Something like: 

```java
protected Properties addProperties() {
  
}
```

Thus it let the user add any property if necessary

Reply:
It is possible to allow users to give properties:
1- Can set a HasMap (key/value). But is not it risky?
2- Another can be to to set value for defined key.